cfme/common/__init__.py

```diff
@@ -155,6 +161,129 @@ def get_tags_db(self):
         return tags
 
 
+class TagPageView(BaseLoggedInPage):
+    """Class represents common tag page in CFME UI"""
+    title = Text('#explorer_title_text')
+    table_title = Text('//div[@id="tab_div"]/h3')
+    tag_table = Table("//div[@id='assignments_div']//table")
+    tag_category = BootstrapSelect(id='tag_cat')
+    tag_name = BootstrapSelect(id='tag_add')
+
+    save_button = Button('Save')
+    cancel_button = Button('Cancel')
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.table_title.text == 'Tag Assignment' and
+            self.tag_table.is_displayed
+        )
+
+
+class WidgetasticTaggable(object):
+    """
+        This class can be inherited by any class that honors tagging.
+        Class should have 'EditTags' navigate step registered.
+        This class should be used for already converted to widgetastic classes
+
+        It provides functionality to assign and unassigned tags.
+    """
+
+    def edit_tag(self, tag, cancel=False, reset=False):
```

Comment:
**OPTIONAL**
I don't like `edit_tag()` has tag argument as a list. Is there any reason to do that? I would do something like that:
```python
def edit_tag(self, category, tag_name, cancel=False, reset=False)
```